<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Edit_Usuario</class>
 <widget class="QFrame" name="Form_Edit_Usuario">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LC INFORMÁTICA | EDITAR USUÁRIOS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../img/edit-.jpg</normaloff>../../img/edit-.jpg</iconset>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="fr_FormProdutos">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>1000</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #FFF;
border: none</string>
   </property>
   <widget class="QLabel" name="lb_FormProdutos">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>10</y>
      <width>961</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 14px;
font-family: &quot;Arial&quot;;
font-weight: bold;
	color: rgb(0, 170, 255);
border-bottom: 2px solid #A2A2A2
}</string>
    </property>
    <property name="text">
     <string>EDITAR USUÁRIO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_idProduto">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border: 1px solid #A2A2A2;
color: #000;
font-size: 14px;
font-family: &quot;Arial&quot;;
font-weight: bold;

}</string>
    </property>
    <property name="text">
     <string>selecione uma foto</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FotoProduto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #A2A2A2;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>NOME COMPLETO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_nome">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>85</y>
      <width>450</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;
text-transform: uppercase;
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="placeholderText">
     <string>DIGITE SEU NOME</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>NÍVEL DE ACESSO</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_Nivel_Acesso">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;
text-transform: uppercase
}
QComboBox:Focus {
border: 1px solid red;
}
 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 25px;
     border-left-width: 1px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }
QComboBox::down-arrow {
     image: url(&quot;+self.resourcepath('Images/down.png')+&quot;);
 }
</string>
    </property>
    <item>
     <property name="text">
      <string>SELECIONE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ADMINISTRADOR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>USUÁRIO</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="bt_AddCategoriaProduto">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>320</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #7AB32E;
color: #FFF
}
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/add/Sem.png</normaloff>:/add/Sem.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>290</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>PERMISSÕES</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_Permissoes">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>320</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;
text-transform: uppercase
}
QComboBox:Focus {
border: 1px solid red;
}
 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 25px;
     border-left-width: 1px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }
QComboBox::down-arrow {
     image: url(&quot;+self.resourcepath('Images/down.png')+&quot;);
 }
</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtBottom</enum>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>SELECIONE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VISUALIZAR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ALTERAR</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="bt_AddMarcaProduto">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>320</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #7AB32E;
color: #FFF
}
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/add/Sem.png</normaloff>:/add/Sem.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>960</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 14px;
font-family: &quot;Arial&quot;;
font-weight: normal;

border-bottom: 2px solid #A2A2A2;
color: #797979
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="fr_BotoesFormProdutos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>1000</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: none;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_qtdeMin">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>350</y>
      <width>40</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_PorcentagemVarejo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #FFF;
border-radius: 2px;
color: #7AB32E;
font: 20px &quot;Arial&quot; ;
font-weight: bold
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_PorcentagemAtacado">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>420</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #FFF;
border-radius: 2px;
color: #7AB32E;
font: 20px &quot;Arial&quot; ;
font-weight: bold
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_CancelAddCatergoria">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cancelar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #7AB32E;
color: #FFF
}
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>NOME DE USUÁRIO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Login">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>450</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;
text-transform: uppercase;
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="placeholderText">
     <string>DIGiTE SEU LOGIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>SENHA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Senha">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;
text-transform: uppercase;
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>DIGITE SUA SENHA</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FormProdutos_9">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font-size: 12px;
font-family: &quot;Arial Unicode MS&quot;;
font-weight: bold;
color: #797979
}</string>
    </property>
    <property name="text">
     <string>REPITA A SENHA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tx_Senha2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background: #CFCFCF;
border-radius: 2px;
color: #000;
font: 13px &quot;Arial&quot; ;
text-transform: uppercase;
}
QLineEdit:Focus {
border: 1px solid red;
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>DIGITE A SENHA NOVAMENTE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecione uma foto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_SalvarUsuario">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>440</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #7AB32E;
color: #FFF
 }
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string>SALVAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/salvar/salvar.png</normaloff>:/salvar/salvar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Bt_CancelarUsuario">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>440</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #1E87F0;
color: #FFF
 }
QPushButton:hover{
background-color: #40a286
}</string>
    </property>
    <property name="text">
     <string>CANCELAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/sair/delete_delete_exit_1577.png</normaloff>:/sair/delete_delete_exit_1577.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <zorder>Bt_CancelAddCatergoria</zorder>
   <zorder>lb_FormProdutos</zorder>
   <zorder>tx_idProduto</zorder>
   <zorder>lb_FotoProduto</zorder>
   <zorder>lb_FormProdutos_2</zorder>
   <zorder>tx_nome</zorder>
   <zorder>lb_FormProdutos_3</zorder>
   <zorder>cb_Nivel_Acesso</zorder>
   <zorder>bt_AddCategoriaProduto</zorder>
   <zorder>lb_FormProdutos_4</zorder>
   <zorder>cb_Permissoes</zorder>
   <zorder>bt_AddMarcaProduto</zorder>
   <zorder>lb_FormProdutos_8</zorder>
   <zorder>fr_BotoesFormProdutos</zorder>
   <zorder>lb_qtdeMin</zorder>
   <zorder>tx_PorcentagemVarejo</zorder>
   <zorder>tx_PorcentagemAtacado</zorder>
   <zorder>lb_FormProdutos_5</zorder>
   <zorder>tx_Login</zorder>
   <zorder>lb_FormProdutos_6</zorder>
   <zorder>tx_Senha</zorder>
   <zorder>lb_FormProdutos_9</zorder>
   <zorder>tx_Senha2</zorder>
   <zorder>pushButton</zorder>
   <zorder>bt_SalvarUsuario</zorder>
   <zorder>Bt_CancelarUsuario</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tx_nome</tabstop>
  <tabstop>tx_Login</tabstop>
  <tabstop>tx_Senha</tabstop>
  <tabstop>tx_Senha2</tabstop>
  <tabstop>cb_Nivel_Acesso</tabstop>
  <tabstop>bt_AddMarcaProduto</tabstop>
  <tabstop>tx_PorcentagemVarejo</tabstop>
  <tabstop>tx_PorcentagemAtacado</tabstop>
  <tabstop>Bt_CancelAddCatergoria</tabstop>
  <tabstop>tx_idProduto</tabstop>
  <tabstop>cb_Permissoes</tabstop>
  <tabstop>bt_AddCategoriaProduto</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../Icones/icones.qrc"/>
 </resources>
 <connections/>
</ui>
